requirements/optional-private.txt

```diff
@@ -3,7 +3,7 @@
 # git config --global url.https://<private hostname>/.insteadOf https://private.repo/
 
 git+https://private.repo/CFPB/agreement_database.git@2.2.7#egg=agreement-database
-git+https://private.repo/CFPB/cfgov-selfregistration.git@v1.2#egg=selfregistration
+selfregistration==@1.3.1
```

Comment:
Actually, @rosskarchner, thinking about this a little more: how will this work with `pip install -r` locally? What configuration do we need to add/document for that use-case?